src/js/tech/tech.js

```diff
@@ -62,7 +69,17 @@ class Tech extends Component {
       this.manualTimeUpdatesOn();
     }
 
-    if (options.nativeCaptions === false || options.nativeTextTracks === false) {
+    ['audio', 'video', 'text'].forEach((type) => {
+      let capitalType = toTitleCase(type);
+
+      if (settings[`native${capitalType}Tracks`] === false) {
+        this[`featuresNative${capitalType}Tracks`] = false;
+      }
+
+      this[`${type}Tracks_`] = settings[`${type}Tracks`];
+    });
+
+    if (settings.nativeCaptions === false) {
```

Comment:
I check nativeCaptions below the loop. Since its sort of a special case and it was in an or previously.